libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+bundle:
+  name: com.mycompany.{{.project_name}}
+
+include:
+  - res/*.yml
```

Comment:
why not `resources` instead of `res`?

Reply:
I personally prefer having everything in one file but I guess having jobs specs external can help with some project folder CUJs

Reply:
+1 for `resources`, I already made that change in my local copy